unity-renderer/Assets/DCLPlugins/ECS7/Systems/SceneBoundsCheckerSystem/ECSSceneBoundsCheckerSystem.cs

```diff
@@ -32,9 +38,36 @@ public class ECSSceneBoundsCheckerSystem
             this.physicsCollidersComponent = physicsColliderComponent;
             this.audioSourceComponent = audioSourceComponent;
 
+            DataStore.i.ecs7.scenes.OnAdded += OnSceneAdded;
+            DataStore.i.ecs7.scenes.OnRemoved += OnSceneRemoved;
```

Comment:
could we save a reference to `DataStore.i.ecs7.scenes` and use that instead of ` DataStore.i.ecs7.scenes` or use dependency injection
whatever you prefer

Reply:
Done, injected